core/services/ocrcommon/config.go

```diff
@@ -1,37 +1,14 @@
 package ocrcommon
 
 import (
-	"time"
-
 	"github.com/pkg/errors"
 
 	"github.com/smartcontractkit/libocr/commontypes"
 
 	"github.com/smartcontractkit/chainlink/v2/core/config"
-	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/ethkey"
-	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/p2pkey"
-	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
 )
 
 type Config interface {
```

Comment:
Is this still useful as just `ChainType()?

Reply:
Yep, it is still used in a handful of disparate places.